e2etest/utils.go

```diff
@@ -50,11 +50,17 @@ func wgetRetry(address string, exc executor.Executor) error {
 	return err
 }
 
-func tweakServicePort(svc *v1.Service) {
+func tweakService(svc *v1.Service, version string) {
 	if servicePodPort != 80 {
 		// if servicePodPort is non default, then change service spec.
 		svc.Spec.Ports[0].TargetPort = intstr.FromInt(int(servicePodPort))
 	}
+
+	if version == "ipv6" {
```

Comment:
are you sure this is needed? I would have said that creating a service in an ipv6 only cluster will create an ip6 service. Also, we have ipv6 tests in place in l2 and this was not required.

Reply:
it seems i don't need